src/components/SEO.jsx

```diff
@@ -43,6 +59,11 @@ export const SEO = ({ title, seoDescription, keywords }) => {
         name="twitter:image"
         content="https://github.com/holtzy/The-Python-Graph-Gallery/blob/master/static/overview_PGG.png?raw=true"
       />
+      <Script
+        id="adthrive"
+        strategy="idle"
```

Comment:
I've set the script to load when the main thread is idle, which is probably overkill. You could try removing this to revert to the default strategy (load after hydration), but since you were having issues with ad dom being unmounted, I wanted to be sure this would work as you required.